### PR TITLE
chore: Changelog for v4.6.0

### PR DESCRIPTION
**Pull request by lgsilva3087**
SaunaFS (4.6.0) (2024-09-05)
 - fix(mount): Set default cacheExpirationTime to 0 (#180)
 - feat(mount): Add readcachemaxsize option (#145)
 - feat(master): Add empty reserved files option (#176)
 - fix(mount): Fix random case option parsing (#135)
 - feat(chunkserver): Add IDiskManager::getDiskForGC (#169)
 - test: Improve `stop_ganesha()` function (#168)
 - fix(ci): Modify timeout from env (#165)
 - ci: Fix tests behavior (#172)
 - fix(uraft): Fix missing floating ip address
 - fix(mount): Fix create time (Windows) (#167)
 - feat(admin): Add list-disk-groups command (#155)

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+ - test: Improve `stop_ganesha()` function (#168)
+ - fix(ci): Modify timeout from env (#165)
+ - ci: Fix tests behavior (#172)
+ - fix(uraft): Fix missing floating ip address
+ - fix(mount): Fix create time (Windows) (#167)
+ - feat(admin): Add list-disk-groups command (#155)
+
  * SaunaFS (4.5.0) (2024-08-12)
  - ci: Abort previous concurrent tests
  - ci: Update de base docker image on build
```
Comment (uristdwarf) [inline NEWS:18]: The 4.5.0 and 4.6.0 could be merged together here

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+  * ci: Fix tests behavior (#172)
+  * fix(uraft): Fix missing floating ip address
+  * fix(mount): Fix create time (Windows) (#167)
+  * feat(admin): Add list-disk-groups command (#155)
+
+ -- SaunaFS Team <contact@saunafs.com>  Thu, 05 Sep 2024 14:23:04 +0000
+
 saunafs (4.5.0) stable; urgency=medium
   * ci: Abort previous concurrent tests
   * ci: Update de base docker image on build
```
Comment (uristdwarf) [inline debian/changelog:18]: Ditto

File: CMakeLists.txt [hunk 3]
```diff
@@ -189,7 +189,7 @@ endif()
 project(saunafs)
 
 if(NOT PACKAGE_VERSION)
-  set(DEFAULT_MIN_VERSION "4.5.0")
```
Comment (uristdwarf) [inline CMakeLists.txt:192]: It should still be 4.5.0, since we haven't released it yet

**Review by uristdwarf — CHANGES_REQUESTED** [review]

**Review by ralcolea — APPROVED** [review]
LGTM

File: NEWS (hunk 1 above)
Comment (lgsilva3087) [reply NEWS:18]: As we have published a release candidate 4.5.0-rc1 in github. And from that one, there are new features, to honor conventional commits and semantic versioning, we decided to use 4.6.0 (which was also suggested by the script after analyzing the changelog).

**Review by uristdwarf — APPROVED** [review]

**Comment by lgsilva3087** [comment]
Not needed.

**Pull request closed without merging**